src/ussd_popi_user_data.js

```diff
@@ -1,3 +1,4 @@
+/*jshint loopfunc:true */
```

Comment:
That suppresses the warning when a promise is made in a loop. But I think I can remove it now actually..